core-plugins/widgets/GroupByAggregate-batchaggregator.json

```diff
@@ -35,17 +35,17 @@
               "alias": "alias"
             },
             "dropdownOptions": [
-              "Avg",
-              "Count",
-              "First",
-              "Last",
-              "Max",
-              "Min",
-              "Stddev",
-              "Sum",
-              "Variance",
-              "CollectList",
-              "CollectSet"
+              "avg",
```

Comment:
these shouldn't change, pretty sure product wanted capitalized options in the UI.

Reply:
Reverted them to the capitalized version and implemented a map in the aggregator to handle the naming mismatch